packages/account/src/Sections/Profile/PhoneVerification/otp-verification.tsx

```diff
@@ -38,19 +43,34 @@ const ConfirmYourEmail = observer(() => {
     return (
         <PhoneVerificationCard is_small_card>
             <Text bold>
-                <Localize i18n_default_text="Confirm it's you" />
+                {should_show_phone_number_otp ? (
```

Comment:
i think instead of depending on  `should_show_phone_number_otp` from ui store, 

we can check the status from BE and show the component based on the verification status.

Reply:
Yeap but for now the API is not ready so using this to mock the scenario of redirecting from `confirm your phone number` component. I'll add a TODO here just to remove the condition once the API is ready 🙏